.yarn/patches/@web3-onboard-core-npm-2.17.0-158dbf89b9.patch

```diff
@@ -8,12 +8,26 @@ index 06ea0c4a094b86395ce012089b34d6f143cc7101..dbd1c739538b41f4c3a06997e735b111
  			create_component(chainbadge.$$.fragment);
 -			attr(div, "class", "absolute chain-icon-container svelte-jvic9v");
 +      attr(div, "class", "absolute chain-icon-container svelte-jvic9v");
-+      attr(div, "part", "sidebar-heading-img");
++
  		},
  		m(target, anchor) {
  			insert(target, div, anchor);
+diff --git a/dist/Index-f8e62c0c.js b/dist/Index-f8e62c0c.js
+index 69aab46f4550d47306095dec9c489e76ae27fcfa..4e7a47eb6a72e0012b515d34530f7a0f8dc432f9 100644
+--- a/dist/Index-f8e62c0c.js
++++ b/dist/Index-f8e62c0c.js
+@@ -1195,7 +1195,8 @@ function create_default_slot(ctx) {
+ 			t6 = space();
+ 			button1 = element("button");
+ 			t7 = text(t7_value);
+-			attr(div0, "class", "icon-container flex justify-center items-center svelte-1ubxcdp");
++      attr(div0, "class", "icon-container flex justify-center items-center svelte-1ubxcdp");
```

Comment:
I don't understand what you want to do here ? to replace class="" prop with mobile-icon-container or to add this class name to the list ?

in first case you need to replace whole line 24 to attr(div0, "class", "mobile-icon-container"), in the second one  - to attr(div0, "class", "mobile-icon-container icon-container flex justify-center items-center svelte-1ubxcdp")

anyway you don't need both 24+25 lines...

OR you just have missed and it should be part="mobile-icon-container" as in the previous change ??

also see my previous comment regarding packages patching

Reply:
The patch is about adding a part property unto the div, this we can target from outside the shadow-dom
the above line is basically shown in patch because of some tabbing